pkg/chart/metadata_test.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
remove indentation

Reply:
@hickeyma I fixed this for this PR. 

It is a bit strange, though, to have part of the indentation with spaces and part with tabs, which is why my IDE changed this automatically: 

![image](https://user-images.githubusercontent.com/1847260/210843758-9849de62-e13d-4d25-b0a7-074a0aeeac95.png)

I noticed that this seems to be something that is not consistent over the codebase. Not sure, if there is some discussion ongoing about this issue somewhere....

Reply:
Not sure if this was ever raised. If you want, you could raise a issue and some examples in it?